src/ListCatalog.js

```diff
@@ -159,6 +213,22 @@ const ListDemos = () => (
       <ListItem lineOne='Potatoes' lineTwo='30 Noc 2017' leadingIcon='folder' trailingIcon='info' />
       <ListItem lineOne='Carrots' lineTwo='17 Oct 2017' leadingIcon='folder' trailingIcon='info' />
     </ListVariant>
+
+    <ListVariant title='List with Trailing Checkbox' avatars>
+      <ListItem lineOne='Dog Photos' trailingCheckbox='true' />
```

Comment:
`trailingCheckbox` is implicitly set to `true` without `='true'`. See `TextFieldCatalog.js`